src/components/datepicker/datePicker.js

```diff
@@ -570,11 +574,10 @@
   /** Close the floating calendar pane. */
   DatePickerCtrl.prototype.closeCalendarPane = function() {
     if (this.isCalendarOpen) {
-      this.isCalendarOpen = false;
       this.detachCalendarPane();
+      this.isCalendarOpen = false;
```

Comment:
I'm not a fan of this because now the person working on this needs to know that `isCalendarOpen` _must_ be set after detaching; this is a weird coupling of behaviors that makes brittle code. It would be much better to simple call `enableScrolling()` in the `$destroy` handler in addition to `detachPane`